pkg/controller/command/didclient/command_test.go

```diff
@@ -269,6 +281,76 @@ func TestCommand_CreateBlocDID(t *testing.T) {
 		}})
 		require.NoError(t, err)
 
+		var b bytes.Buffer
+		cmdErr := c.CreateTrustBlocDID(&b, bytes.NewBuffer(r))
+		require.NoError(t, cmdErr)
+
+		docRes, err := did.ParseDocumentResolution(b.Bytes())
+		require.NoError(t, err)
+		require.NotEmpty(t, docRes)
+		require.Equal(t, "did:peer:21tDAKCERh95uGgKbJNHYp", docRes.DIDDocument.ID)
+	})
+
+	t.Run("test success create did with Ed25519 p384 key", func(t *testing.T) {
```

Comment:
a p384 is not an ed25519 key.

Reply:
Did you mean with ed25519, p256 and p384 keys?

Reply:
fixed